js/interface.js

```diff
@@ -154,6 +154,14 @@ var DynamicLists = (function() {
           listLayout = $(this).data('layout');
           isLayoutSelected = true;
 
+          Fliplet.Studio.emit('page-preview-send-event', {
+            type: 'layout',
```

Comment:
Event types are shared across all apps and features so we should keep this specific to the LFD.

```suggestion
            type: 'dynamicListLayout',
```